30-remote-site-setup.sh

```diff
@@ -55,6 +55,9 @@ Host $remote_fqdn
   Port $remote_port
   IdentityFile ${ssh_key}
   IdentitiesOnly yes
+  ControlMaster auto
+  ControlPath ~/.ssh/cm-%r@%h:%p
+  ControlPersist  15m
```

Comment:
Interesting, i didn't know about the (new?) ControlPersist option.

Is this just for a speedup then, since an ssh key file is already being used for (passwordless?) auth?

I do enjoy me some ssh session multiplexing...

Reply:
Nope; there's a site that rate limits SSH connections; we're hoping this will count as a single connection so we get around that.

Reply:
Oh cool, that sounds like a great use case